src/main/app/case/case.ts

```diff
@@ -547,7 +547,8 @@ export interface Case {
   miam_childProtectionEvidence?: string[];
   miam_urgency?: string[];
   miam_previousAttendance?: string[];
-  miam_notAttendingReasons?: string[];
+  miam_notAttendingReasons?: string;
+  miam_noMediatorReasons?: string;
```

Comment:
Please create necessary enums for all the miam fields and have strings only for free text areas.